dev/bots/analyze_sample_code.dart

```diff
@@ -166,11 +201,14 @@ class SampleChecker {
 
   /// The temporary directory where all output is written. This will be deleted
   /// automatically if there are no errors.
-  Directory _tempDirectory;
+  final Directory _tempDirectory;
 
   /// The package directory for the flutter package within the flutter root dir.
   final Directory _flutterPackage;
 
+  /// The package directory for the flutter package within the flutter root dir.
```

Comment:
maybe document what it means if this is null?

Reply:
Done. And added some doc comments to the constructor for the parameters.